chapters/3-no-name.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: 'No Name'
+title: No Name
```

Comment:
I'm curious to learn why the quotes were removed here, but not on others (like Chapter 27). Is there a pattern I should learn?

Reply:
oops, I started doing it for no real reason, and just didn't finish. Both work fine.